interval_dist: 5.0
pose_topic: '/amcl_pose'

